src/browser/modules/DBMSInfo/MetaItems.jsx

```diff
@@ -64,7 +65,7 @@ const createItems = (
   if (showStar) {
     let str = '*'
     if (count) {
-      str = `${str}(${count})`
+      str = `${str}(${thousandsCommaSeparatedNumber(count)})`
```

Comment:
I did this here as I assume we would always want to comma separate whenever `createItems` is called...